esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -5703,11 +5698,20 @@ bool CWsWorkunitsEx::onWUEclDefinitionAction(IEspContext &context, IEspWUEclDefi
         if (target.trim().isEmpty())
             throw MakeStringException(ECLWATCH_INVALID_INPUT,"Target not defined in onWUEclDefinitionAction.");
 
+        int msToWait = req.getMsToWait();
+        if (msToWait == -2) //default: consistent with WUPublishWorkunit/WUDeployWorkunit/WUSyntaxCheckECL
```

Comment:
@wangkx Where in the code is Syntax check and publish setting these defaults if -1?  I can't find it to compare.

Reply:
Oh I see you meant the ESDL defaults for those services.  I don't think you need to match those.  Either default to the larger value, or default to -1 (wait forever).